docs/mv3-architecture.md

```diff
@@ -0,0 +1,50 @@
+<!--
+Copyright (c) Microsoft Corporation. All rights reserved.
+Licensed under the MIT License.
+-->
+
+## Accessibility Insights Manifest v3 Extension Architecture
+
+This document describes the top level architecture for the Accessibility Insights extension. It was written after completing the transition from chrome's [manifest v2](https://developer.chrome.com/docs/extensions/mv2/) to [manifest v3](https://developer.chrome.com/docs/extensions/mv3/), and as a result it highlights the changes made during that transition.
+
+### Extension Contexts
+
+At a top level, the extension has 4 separate pages/ operating contexts. These separate environments communicate via flux messaging, as described [in the flux section](#flux).
```

Comment:
nit: It may also be worth mentioning the Devtools and/or DebugTools contexts, at least briefly